pkg/reconciler/instances/connectivityproxy/action.go

```diff
@@ -142,6 +142,10 @@ func (a *CustomAction) Run(context *service.ActionContext) error {
 		return errors.Wrap(err, "Error during reconciliation")
 	}
 
+	if err := a.Commands.PatchConfigMap(context, "kyma-system", "connectivity-proxy"); err != nil {
```

Comment:
This function should not be called every reconcilation.  PatchConfigMap  must be done only if we are upgrading from 2.4.4 to 2.9.2. 

```suggestion
         if (refresh) {
                 if currentVersion == 2.4.4 && installedVersion == 2.9.2 {
	                if err := a.Commands.PatchConfigMap(context, "kyma-system", "connectivity-proxy"); err != nil {
	        }
	}
```

Reply:
Done